wikimedia/wikipedia-ios
# Dependency updates

**Fixes Phabricator ticket:** N/A

### Notes
* Removes Tweaks, as I don't think we use it anymore. (Notably, we still have exists because the our NYTimesPhotoViewer version uses it.)
* Updates LibrariesUsed info shown in app.
* Next up - updating our forks of a few libraries.

### Test Steps
1. Run app. Ensure it works. 
2. Go to Settings, ensure it works.
